### PR TITLE
remove-undefined-body-in-response

### PR DESCRIPTION
**Pull request by qiaozha**
fixes https://github.com/Azure/autorest.typescript/issues/1771

### CODE REVIEW

File: packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/responses.ts [hunk 1]
```diff
@@ -169,8 +167,6 @@ export interface AttachmentsDeleteDefaultResponse extends HttpResponse {
 /** Downloads and returns attachment as response for the given input filePath. */
 export interface AttachmentsDownload200Response extends HttpResponse {
   status: "200";
-  /** Value may contain any sequence of octets */
-  body: Uint8Array;
```
Comment (MaryGao) [inline packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/responses.ts:173]: Why we remove this? Not sure it's possible to add UT.

**Review by MaryGao — APPROVED** [review]
LGTM, btw could you check if this exists in TypeSpec ?

File: packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/responses.ts (hunk 1 above)
Comment (qiaozha) [reply packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/responses.ts]: it's deleted by mistake, added back.

**Comment by qiaozha** [comment]
> LGTM, btw could you check if this exists in TypeSpec ?

Checked, should not exist on the typespec side.

**Pull request merged**